docs/how_to_contribute/math/how_to_add_prime_fields.md

```diff
@@ -8,7 +8,21 @@ _Note_: We have our own development conventions. Please read the [conventions do
 
 Begin by creating a directory named `<new_prime_field>_prime` in `/tachyon/math/finite_field/`. Add a `BUILD.bazel` file into this directory. Note that once parameters are added to `BUILD.bazel`, Bazel will automatically generate the prime field code based on these parameters when it builds the target.
 
-Choose among `generate_prime_fields()`, `generate_fft_prime_fields()`, or `generate_large_fft_prime_fields()` for code generation, depending on the prime field type. For more information, refer to [`prime_field_generator`](/tachyon/math/finite_fields/generator/prime_field_generator/build_defs.bzl).
+### [Prime field generator]((/tachyon/math/finite_fields/generator/prime_field_generator/build_defs.bzl))
+
+Choose the prime field generator depending on the prime field type.
```

Comment:
Maybe instead of saying "In addition to the properties of ..." repetitively for generator 2/3, you can start out by saying something along the lines of "Note that each successive field type in this list additionally requires all properties mentioned in previous field types."